index.js

```diff
@@ -1,39 +1,342 @@
+          global.game.moveToRoom('Foyer'); // move to desired room
+          inRoom = false;
+        } else {
+          console.log(`BZZZTT wrong code`);
+        }
+      } else if (input.includes("mr mikes") || input.includes("eat") || input.includes("drink") && input.includes("beer")){
+        console.log('You go eat and grab a beer at Mr. Mikes. You are full and happy');
+        global.game.user.changeStatus('Full and Happy');
+        process.exit();
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how to ${input}`);
+      }
+    }
+  } // end while
+
+  play(global.game.currentRoom);
+}
+
+// foyer() {}
+async function foyer() {
+  console.log(global.game.currentRoom.message);
+  let inRoom = true;
+
+  // write Foyer logic block
+  while (inRoom) {
+    console.log(`Player Status: ${global.game.user.checkStatus()}`);
+    let input = await ask('\n>_');
+    input = input.toLowerCase();
+
+    //check for common commands
+    if (commonCommands(input)) {
+      // do nothing, because a common command was found
+    } else {
+      // take the seven days paper
+      if ( input.includes("take") && (input.includes("paper") || input.includes("seven days")) ) {
 
-async function start() {
-  const welcomeMessage = `182 Main St.
-You are standing on Main Street between Church and South Winooski.
-There is a door here. A keypad sits on the handle.
-On the door is a handwritten sign.`;
-  let answer = await ask(welcomeMessage);
-  console.log('Now write your code to make this work!');
-  process.exit();
+        if (!global.game.user.inventory.find(item => item === "seven days")) {
+          global.game.user.addToInventory("seven days");
+        } else {
+          console.log("You already picked that up");
+        }
+        global.game.user.showInventory();
+        global.game.currentRoom.showInventory();
+      
+        // drop the seven days paper
+      } else if ( input.includes("drop") && (input.includes("paper") || input.includes("seven days")) ) {
+
+        if (global.game.user.inventory.find(item => item === "seven days")) {
+          global.game.user.dropItem("seven days");
+        } else {
+          console.log("You don't have that item!");
+        }
+        global.game.user.showInventory();
+        global.game.currentRoom.showInventory();
+      } else if (input.includes("go") && input.includes("upstairs")) {
+        global.game.moveToRoom('Classroom');
+        inRoom = false;
+      } else if (input.includes("go") && input.includes("outside")){
+        global.game.moveToRoom('Main');
+        inRoom = false;
+        // catch-all for unrecognized input
+      } else {
+        console.log(`Sorry I don't know how tolet compareSimilarity = (input, triggerWords) => {
+${input}`);
+      }
+    }
+  } // end while
+  // go to the next room
+  play(global.game.currentRoom);
 }
+
+
+async function classroom() {
```

Comment:
All your room functions could probably be combined into one generic 'game' function with a little refactoring, and some expansion of the `Room` class. This would allow you to remove 40-50 lines of code, and would make it much easier to scale up the program